Objects/lnotab_notes.txt

```diff
@@ -45,7 +45,8 @@ above example contains two such values. So we make two tweaks:
             line_incr -= 0x100
         lineno += line_incr
 
-(In C, this is implemented by PyCode_Addr2Line().)  In order for this to work,
+(In C, this is implemented by PyCode_Addr2Line(). Note that before 3.6, only
```

Comment:
Use two spaces between sentences.

I think that the better place for this note is few lines above, when describe the range of `line_incr`.